ibis/expr/datatypes.py

```diff
@@ -526,6 +526,13 @@ def from_tuples(
         names, types = zip(*pairs)
         return cls(list(names), list(map(dtype, types)), nullable=nullable)
 
+    @classmethod
+    def from_dict(
```

Comment:
may want to consider dispatching in the future, like: https://github.com/ibis-project/ibis/blob/master/ibis/expr/schema.py#L172

Reply:
Ohoh good point!